HLTrigger/Configuration/python/HLT_75e33/modules/hltIter2Phase2L3FromL1TkMuonPixelSeedsFiltered_cfi.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+hltIter2Phase2L3FromL1TkMuonPixelSeedsFiltered = cms.EDProducer("MuonHLTSeedMVAClassifierPhase2",
+    src    = cms.InputTag("hltIter2Phase2L3FromL1TkMuonPixelSeeds"),
+    L1TkMu = cms.InputTag("l1tTkMuonsGmt"),
+    mvaFile_B_0 = cms.FileInPath("RecoMuon/TrackerSeedGenerator/data/xgb_Phase2_Iter2FromL1_barrel_v0.xml"),
+    mvaFile_E_0 = cms.FileInPath("RecoMuon/TrackerSeedGenerator/data/xgb_Phase2_Iter2FromL1_endcap_v0.xml"),
+    mvaScaleMean_B = cms.vdouble(0.00033113700731766336, 1.6825601468762878e-06, 1.790932122524803e-06, 0.010534608406382916, 0.005969459957330139, 0.0009605022254971113, 0.04384189672781466, 7.846741237608237e-05, 0.40725050850004824, 0.41125151617410227, 0.39815551065544846),
```

Comment:
Hi, These values are used to standardize the input variables of BDT. This values were generated by scikit-learn's StandardScaler, and I haven't tested how much it will affect the model's performances if I change the value. Run3 seed classifier menu also keeps these values with 17 digits, so I thought following that would be safe.

Reply:
OK, would still be interesting to know if we really need to drag around all these digits, so maybe you could study the performance varying the number of digits post-merge.